src/components/AutoSuggest/AutoSuggest.tsx

```diff
@@ -351,15 +351,14 @@ const AutoSuggest = ({
 
   return (
     <Wrapper className={className} ref={wrapperRef} data-testid="auto-suggest">
-      <div
-        aria-controls="as-listbox"
-        aria-expanded={showList}
-        aria-haspopup="listbox"
-        role="combobox"
-      >
+      <div>
         <Input
           autoFocus={autoFocus}
           aria-activedescendant={activeId.toString()}
+          aria-expanded={showList}
+          aria-haspopup="listbox"
+          role="combobox"
+          aria-controls="as-listbox"
```

Comment:
One more suggestion from Robbert Broersma (NL Design System) is to add a `spellcheck="false"` here.

Reply:
Fixed :)